samples/poc-subapp/src/06.suspense-demo/subapp-suspense-demo.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { loadSubApp } from "subapp-web";
+import PropTypes from "prop-types";
+
+let data;
+
+class Data1 extends React.Component {
+  constructor() {
+    super();
```

Comment:
Just curious, why are you overriding the constructor in these demos? Is it intentionally clearing out the construction props? If not, why not pass the props to `super`?

Reply:
good catch. :) thanks.

Reply:
this change actually seem to got mixed up with another PR.  ahh... trying to juggle 5 PRs with git. :)